frontend/src/model/beta/Settings.tsx

```diff
@@ -1,21 +1,34 @@
 import { Box, Button, Divider, List, ListItem, ListItemButton, Stack, Typography } from '@mui/material'
-import { useState } from 'react'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
 
 import { ModelInterface } from '../../../types/v2/types'
 import AccessRequestSettings from './settings/AccessRequestSettings'
 import ModelAccess from './settings/ModelAccess'
+import ModelDetails from './settings/ModelDetails'
 
-type SettingsCategory = 'general' | 'danger' | 'access'
+type SettingsCategory = 'details' | 'danger' | 'access' | 'permissions'
 
 type SettingsProps = {
   model: ModelInterface
 }
 
 export default function Settings({ model }: SettingsProps) {
-  const [selectedCategory, setSelectedCategory] = useState<SettingsCategory>('general')
+  const router = useRouter()
+
+  const { section } = router.query
+
+  const [selectedCategory, setSelectedCategory] = useState<SettingsCategory>('details')
+
+  useEffect(() => {
+    section ? setSelectedCategory(section as SettingsCategory) : setSelectedCategory('details')
```

Comment:
This type assertion is unsafe, you should use a type guard to make sure this is okay.  This line can also be replaced with:

```
setSelectedCategory(section ?? 'details')
```

Using the null coalescing operator.